src/griffe/dataclasses.py

```diff
@@ -1392,6 +1415,29 @@ def parameters(self) -> Parameters:
                 )
             return Parameters()
 
+    @cached_property
+    def resolved_bases(self) -> list[Object]:
+        """Resolved class bases."""
+        resolved_bases = []
+        for base in self.bases:
+            base_path = self.resolve(base) if isinstance(base, str) else base.full
+            try:
+                resolved_base = self.modules_collection[base_path]
+                if resolved_base.is_alias:
+                    resolved_base = resolved_base.final_target
+            except (AliasResolutionError, CyclicAliasError, KeyError):
+                logger.debug(f"Base class {base_path} is not loaded, or not static, it cannot be resolved")
+            else:
+                resolved_bases.append(resolved_base)
+        return resolved_bases
+
+    def mro(self) -> list[Class]:
```

Comment:
We could consider caching the result. We do that a lot throughout the code base though, and already consume a lot of memory. We'll need to optimize at some point.